### ISSUE TITLE
ci(custom-checks): fix sibling pkg dependency version inconsistencies

### ISSUE THREAD

**Issue by rwat17** [hyperledger/cacti#2431]
>npm run custom-checks 
In root dir prints these errors:

ERROR: packages/cactus-plugin-ledger-connector-ubiquity/package.json dependencies.@hyperledger/cactus-core incorrectly has version 1.1.2. Expected 1.1.3
ERROR: packages/cactus-plugin-ledger-connector-ubiquity/package.json dependencies.@hyperledger/cactus-core-api incorrectly has version 1.1.2. Expected 1.1.3
ERROR: packages/cactus-plugin-ledger-connector-ubiquity/package.json devDependencies.@hyperledger/cactus-common incorrectly has version 1.1.2. Expected 1.1.3
ERROR: packages/cactus-plugin-ledger-connector-ubiquity/package.json devDependencies.@hyperledger/cactus-test-tooling incorrectly has version 1.1.2. Expected 1.1.3

Are these versions intentionally older for now or should I replace them? 

https://github.com/hyperledger/cacti/blob/main/packages/cactus-plugin-ledger-connector-ubiquity/package.json

https://github.com/hyperledger/cacti/blob/3e6d908cb4cf732ef057a30df3e8267381f3f806/packages/cactus-plugin-ledger-connector-ubiquity/package.json#L58

**Comment by petermetz** [comment]
@rwat17 Thank you for reporting! This is indeed a bug, the versions should be consistent across the framework codebase!
The custom-checks are planned to be part of the CI as well so that every PR that gets the versions out of sync would fail the checks and force us to make them consistent before the diff makes it to the main branch.

TLDR: Yes, please send a PR with making the versions consistent (or let me know if you'd prefer to not work on it and we'll get it done)

**Comment by rwat17** [comment]
@petermetz will do this in the meantime today :)
